docs/source/examples/Widget Custom.ipynb

```diff
@@ -120,7 +120,26 @@
     "\n",
     "You also need to enable the widget frontend extension.\n",
     "\n",
-    "If you are using JupyterLab 2.x:\n",
+    "If you are using JupyterLab 3.x:\n",
+    "\n",
+    "\n",
+    "```bash\n",
+    "# link your development version of the extension with JupyterLab\n",
+    "jupyter labextension develop . --overwrite\n",
+    "\n",
+    "# rebuild extension Typescript source after making changes\n",
+    "jlpm run build\n",
```

Comment:
Perhaps:
```suggestion
    "yarn run build\n",
```
to be fully consistent with the cookiecutter's recommendations? Though this would also require modifying the environment creation command.

Reply:
Thanks for spotting this :+1: 

Then yes let's stay consistent with the cookiecutter and use `yarn`.